aea/cli/add.py

```diff
@@ -48,18 +48,27 @@ def add(ctx: Context, registry):
     try_to_load_agent_config(ctx)
 
 
-def _copy_package_directory(ctx, package_path, item_type, item_name):
+def _is_item_present(item_type, item_public_id, ctx):
+    item_type_plural = item_type + 's'
+    dest_path = os.path.join(ctx.cwd, "vendor", item_public_id.author, item_type_plural, item_public_id.name)
+    items_in_config = getattr(ctx.agent_config, item_type_plural)
+    return item_public_id in items_in_config and os.path.exists(dest_path)
+
+
+def _copy_package_directory(ctx, package_path, item_type, item_name, author_name):
     # copy the item package into the agent's supported packages.
     item_type_plural = item_type + "s"
     src = str(package_path.absolute())
-    dest = os.path.join(ctx.cwd, item_type_plural, item_name)
+    dest = os.path.join(ctx.cwd, "vendor", author_name, item_type_plural, item_name)
     logger.debug("Copying {} modules. src={} dst={}".format(item_type, src, dest))
     try:
         shutil.copytree(src, dest)
     except Exception as e:
         logger.error(str(e))
         sys.exit(1)
 
+    Path(ctx.cwd, "vendor", author_name, item_type_plural, "__init__.py").touch()
```

Comment:
add comment as previously: # make the 'item_type_plural' folder a Python package.